laliga/contracts/Golazos.cdc

```diff
@@ -134,31 +134,35 @@ pub contract Golazos: NonFungibleToken {
 
     /// A public struct to access Series data
     ///
-    pub struct SeriesData {
-        pub let id: UInt64
-        pub let name: String
-        pub let active: Bool
+    access(all) struct SeriesData {
+        access(all) let id: UInt64
+        access(all) let name: String
+        access(all) let active: Bool
 
         /// initializer
         //
-        init (id: UInt64) {
-            let series = (&Golazos.seriesByID[id] as! &Golazos.Series?)!
-            self.id = series.id
-            self.name = series.name
-            self.active = series.active
+        view init (id: UInt64) {
+            if let series = &Golazos.seriesByID[id] as &Golazos.Series?{
```

Comment:
We can add the panic message without if/else block (same in a couple other places on the contract):

```
let series = &Golazos.seriesByID[id] as &Golazos.Series?
   ?? panic("series does not exist")
```

Reply:
done